src/VaxClient.cs

```diff
@@ -4,26 +4,29 @@ namespace Vendasta.Vax
 {
     public class VaxClient
     {
-        private readonly float _defaultTimeout;
+        private readonly RequestOptions _defaultRequestOptions;
 
-        public VaxClient(float defaultTimeout)
+        public VaxClient(float timeout = 10000, bool includeToken = true, RetryOptions retryOptions = null)
```

Comment:
is this a breaking change? I cant remember if defaults let you skip values or not while calling a function/method

Reply:
Non breaking

Reply:
The primary reason for using defaults is so you can add a new parameter without it being a breaking change.